edbdeploy/cli.py

```diff
@@ -49,11 +53,12 @@ class EFMVersionOption:
 
 
 class PgTypeOption:
-    choices = ['PG', 'EPAS']
+    choices = ['PG', 'EPAS', 'RDS']
```

Comment:
Technically, RDS can be publicly available so it could be accessible with gcloud and azure.

But I don't think that's possible with how the deployment tool is implemented.

Another solution isn't immediately obvious to me.  Suggestions?

Reply:
What scenario do you have in mind that could mix RDS and gcloud/azure?

Reply:
The only practical one is migrating databases between clouds.